### ISSUE TITLE
Wallet amount should show all tokens by default

### ISSUE THREAD

**Issue by kingsleyh** [SushiChain/SushiChain#146]
I think wallet amount with no specified token should show a list of all tokens for the address. currently you get only shari by default - and if you want to see token amounts you have to set --token=TOKEN. However most people I think will want to see everything they have at once. If you don't remember what tokens you have or if you get some airdropped to you - it's pretty hard to figure out what you have.

in summary:
```
sushi wt amount => list all tokens with amounts
sushi wt amount --token=KINGS => get amount only for KINGS token
```

**Comment by kingsleyh** [comment]
hmm this is now doing as expected. I think perhaps there is some timing that I don't understand. I did the following:

1. create a new token
2. check the wallet -u (only showed SHARI)
3. transfer 5000 token to another address
4. check both wallets -u - (only showed SHARI and no token)
5. after I checked back much later - it was showing the SHARI and the token

maybe it needed some blocks to be processed before it all started working. Although I would have thought the created tokens should have shown up on the creators wallet as -u as soon as the transaction was added to the block

**Comment by kingsleyh** [comment]
ok - I tried a few more times and I can't re-produce my original problem - I'll re-open if I spot it again
